marklogic-data-hub-central/ui/e2e/hc-qa-project/build.gradle

```diff
@@ -49,3 +52,31 @@ task loadThesaurus(type: com.marklogic.gradle.task.MlcpTask) {
 }
 
 hubDeployAsDeveloper.finalizedBy("loadThesaurus")
+
+class DocCount extends HubTask {
+    DatabaseClient client
+    String script
+    @TaskAction
+    def getDocCount() {
+        try {
+            int result = Integer.parseInt(client.newServerEval().javascript(script).evalAs(String.class));
+            if(result != 34) {
+                throw new GradleException("Record count did not match. Expected 34 in ${client.getDatabase()} database, got "+ result)
+            } else {
+                println("Flow/steps were run successfully and got expected number of records in ${client.getDatabase()} database")
+            }
+        } finally {
+            client.release()
```

Comment:
The task with typo was still passing. Added finally and now both the tasks fail when expected. to